ic-os/components/BUILD.bazel

```diff
@@ -49,6 +49,8 @@ REPO_COMPONENTS = glob(
 ignored_repo_components = [
     "networking/dev-certs/canister_http_test_ca.key",
     "networking/dev-certs/root_cert_gen.sh",
+    "misc/guestos-recovery/guestos-recovery-engine/guestos-recovery-engine.sh",
```

Comment:
I thought it was strange that we'd need to include this, since we _will_ have a variant that uses this, but it is because this will be the first time we dynamically change `component_files` for a variant.

Which had me thinking, should we?

Reply:
> and no other place in the repo seems proper for it.

I agree, to me, ideally we would keep these outside of the repo so there is no way to confuse this as something in the IC (and then it could be committed into a branch if ever needed), but I know that doesn't make much sense for testing. I think we should re-home the upgrader outside of HostOS at some point too, but this is also not the time.

We could move these to `ic-os/recovery`, but I'm fine to leave it as is, as well.